src/api/app/views/webui2/webui/cloud/upload_jobs/_breadcrumb_items.html.haml

```diff
@@ -1,4 +1,10 @@
 = render partial: 'webui/main/breadcrumb_items'
 
-%li.breadcrumb-item.active{ 'aria-current' => 'page' }
-  Cloud Upload
+- if action_name == 'index'
+  %li.breadcrumb-item.active{ 'aria-current' => 'page' }
+    Cloud Upload
+- if action_name == 'new'
```

Comment:
A `case` or `if/else` would be better here to avoid checking this line when `action_name` is `index`.

Reply:
done